certification/internal/policy/operator/deployable_by_olm.go

```diff
@@ -294,6 +294,7 @@ func (p *DeployableByOlmCheck) installedCSV(ctx context.Context, operatorData Op
 		for {
 			log.Debug("Waiting for Subscription.status.installedCSV to become ready...")
 			subs, _ := p.OpenshiftEngine.GetSubscription(ctx, operatorData.App, operatorData.InstallNamespace)
```

Comment:
Is there a case where subs can be nil here ?

Reply:
The check returns if API server fails to create the subscription. So, I would say the subscription exists when we get here.